src/components/SettingComponent.js

```diff
@@ -170,13 +169,16 @@ PopoverColorPicker.propTypes = {
 PopoverColorPicker.defaultProps = { storageKey: null }
 
 export const Checkbox = ( { name, value, storageKey, icon, checkedIcon, onChange, ...props } ) => (
-  <MaterialCheckbox
-    icon={( <FontAwesomeIcon icon={icon} /> )}
-    checkedIcon={( <FontAwesomeIcon icon={checkedIcon} /> )}
-    checked={JSON.parse( value )}
-    onChange={( { target: { checked } } ) => onChange( checked )}
-    {...props}
-  />
+  <div>
+    <MaterialCheckbox
+      icon={( <FontAwesomeIcon icon={icon} /> )}
+      checkedIcon={( <FontAwesomeIcon icon={checkedIcon} /> )}
+      checked={JSON.parse( value )}
+      onChange={( { target: { checked } } ) => onChange( checked )}
+      size=","
```

Comment:
huh?

Reply:
But genuinely, what does `size=","` do?

Reply:
No clue. Did I add it ?

Reply:
My bad I have no clue why I did that, as far as I think “size” is to change the size for the checkbox so that they look the same as color picker but somehow the value that should be in it got replaced with a “,”

Reply:
No worries, I've sorted it